tap/history.ipynb

```diff
@@ -0,0 +1,37 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "bf4b0ab6-2946-444b-a208-fe8821ea353a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from lsst.rsp import get_query_history\n",
+    "hist=await get_query_history()\n",
+    "hist"
+   ]
+  }
+ ],
```

Comment:
Oh I just realized you'll still need an empty code cell before this for Times Square to control, even if there aren't any template values.

Reply:
Thanks!  Just did that.